docs/Using-Fleet/Permissions.md

```diff
@@ -12,45 +12,43 @@ Users with the Admin role receive all permissions.
 | Transfer hosts between teams\*                       |          | ✅         | ✅    |
-| Create saved queries                                 |          | ✅         | ✅    |
-| Edit saved queries                                   |          | ✅         | ✅    |
-| Delete saved queries                                 |          | ✅         | ✅    |
-| Schedule queries for all hosts                       |          | ✅         | ✅    |
-| Schedule queries for all hosts assigned to a team\*  |          | ✅         | ✅    |
-| Create packs                                         |          | ✅         | ✅    |
-| Edit packs                                           |          | ✅         | ✅    |
-| Delete packs                                         |          | ✅         | ✅    |
-| Create labels                                        |          | ✅         | ✅    |
-| Edit labels                                          |          | ✅         | ✅    |
-| Delete labels                                        |          | ✅         | ✅    |
-| Add policies for all hosts                           |          | ✅         | ✅    |
-| Remove policies for all hosts                        |          | ✅         | ✅    |
-| Add policies for hosts assigned to team\*            |          | ✅         | ✅    |
-| Remove policies for hosts assigned to team\*         |          | ✅         | ✅    |
-| Create users                                         |          |            | ✅    |
-| Edit users                                           |          |            | ✅    |
-| Delete users                                         |          |            | ✅    |
-| Edit own user information                            | ✅       | ✅         | ✅    |
+| Create, edit, and Delete labels                      |          | ✅         | ✅    |
+| Browse all software                                  | ✅       | ✅         | ✅    |
+| Filter software by vulnerabilities                   | ✅       | ✅         | ✅    |
+| Filter hosts by software                             | ✅       | ✅         | ✅    |
+| Filter software by team*                             | ✅       | ✅         | ✅    |
+| Manage software by vulnerability automations         |          | ✅         | ✅    |
+| Run saved queries as live queries against all hosts  | ✅       | ✅         | ✅    |
+| Run custom queries as live queries against all hosts |          | ✅         | ✅    |
+| Create, edit, and delete queries                     |          | ✅         | ✅    |
+| Create, edit, and delete scheduled queries for all hosts |          | ✅         | ✅    |
+| Create, edit, and delete scheduled queries for all hosts assigned to a team\*  |          | ✅         | ✅    |
+| Create, edit, and delete packs                       |          | ✅         | ✅    |
+| Browse all policies                                  | ✅       | ✅         | ✅    |
```

Comment:
```suggestion
| View all policies                                  | ✅       | ✅         | ✅    |
```
What do you think about using the word "View" instead of "Browse?" This seems to be more consistent with other applications' permissions docs (ex. [GitLab](https://docs.gitlab.com/ee/user/permissions.html)).

If you agree, can you please make sure that we have "View" permissions included for other entities? (queries, users, etc.)